### PR TITLE
time zone added to timestamp of quarantined

### PR DESCRIPTION
**Pull request by ux-AylonS**
Signed-off-by: AylonS <aylons@reblaze.com>

### CODE REVIEW

File: src/components/QuarantinedList.vue [hunk 1]
```diff
@@ -88,7 +88,11 @@ export default defineComponent({
           isSortByOriginalValue: true,
           displayFunction: (item: any) => {
             const newDate = new Date(item['timestamp'])
-            return DateTimeUtils.isoToNowCuriefenseFormat(newDate)
+            const newDateMinutes = newDate.getTime()
```
Comment (Aviv-Galmidi) [inline src/components/QuarantinedList.vue:91]: `getTime()` returns time in milliseconds, not minutes. Looks like the functionality is ok so just the name of the variable is misleading

File: src/components/QuarantinedList.vue [hunk 2]
```diff
@@ -88,10 +88,10 @@ export default defineComponent({
           isSortByOriginalValue: true,
           displayFunction: (item: any) => {
             const newDate = new Date(item['timestamp'])
-            const newDateMinutes = newDate.getTime()
-            const timeZoneDifference = newDate.getTimezoneOffset()
-            const timeZoneMinutes = timeZoneDifference * 60 * 1000
-            const finalDate = new Date(newDateMinutes - timeZoneMinutes)
+            const newDateMilliSeconds = newDate.getTime()
+            const timeZoneDifferenceMinutes = newDate.getTimezoneOffset()
+            const timeZoneMilliSeconds = timeZoneDifferenceMinutes * 60 * 1000
```
Comment (Aviv-Galmidi) [inline src/components/QuarantinedList.vue:93]: That's minutes now
milliseconds * 1000 = seconds
seconds * 60 = minutes

Comment (ux-AylonS) [reply src/components/QuarantinedList.vue:93]: minutes x 60 x 1000 = milliseconds

Comment (Aviv-Galmidi) [reply src/components/QuarantinedList.vue:93]: Nvm, I got the numbers confused, that's indeed in milliseconds

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**